ACadSharp/Objects/MultiLeaderAnnotContext.cs

```diff
@@ -260,18 +263,25 @@ public partial class MultiLeaderAnnotContext : CadObject
 
 		//	These fields read from DWG are stored into the
 		//	Normal and Location property (see above).
-		//3BD		14		Normal vector
-		//3BD		15		Location
+		[DxfCodeValue(14, 24, 34)]
+		public XYZ BlockContentNormal { get; set; }
 
+		/// <summary>
+		/// Location
+		/// </summary>
+		[DxfCodeValue(15, 25, 35)]
+		public XYZ BlockContentLocation { get; set; }
 		/// <summary>
 		/// Scale vector
 		/// </summary>
 		[DxfCodeValue(16)]
 		public XYZ BlockContentScale { get; set; }
```

Comment:
Are the DXF values 16, 26, 36? if that's the case please add them.

Reply:
Group code is 16 in Open Design Spec DWG and in the DXF reference. But it is clearly a 3BD named Scale Vector.
DXF files contain 16, 26, 36. So this is what is schould be.

BTW: This property of `MultiLeaderAnnotationContext` correponds to the `MultiLeaderStyle` property `BlockContentScale` with the group codes 47, 49, 140, see comment.
_Should we really slpit the `MultiLeaderStyle.BlockContentScale` property and `MultiLeaderAnnotationContext.BlockContentScale` as well?_ (see above).

Reply:
knowing the DXF documentation this codes 47, 49, 140 do not belong to an XYZ component, so I would change them to make it easier for the DXF reader, or if they are directly corelated to the `MultiLeaderStyle` assigned, then they should be read-only for the context and return the right value.

That would be my approach, but this will not be relevant until the dxf reader/writer are in place, I don't mind to keep the current implementation with the note, I would add another note to know the relationship with the Style.

Reply:
> knowing the DXF documentation this codes 47, 49, 140 do not belong to an XYZ component

right.

This property of `MultiLeaderStyle` ist the standard scale vector for block content. `MultiLeaderAnnotationContext` has the same property , that overrides the standard values defined in `MultiLeaderStyle` when the respective `MultiLeaderPropertyOverrideFlag` is set. This property has the dxf-codes 16, 26, 36, that definitely indicate an XYZ-type.

So I would prefer to keep both properties as XYZ.
Maybe we should just now keep it as ist is and discuss it later.